docs/dbus/manager.xml

```diff
@@ -255,6 +255,141 @@
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Set a Stratis key in the kernel keyring. -->
+    <method name="SetKey">
+      <!-- Key description to assign to the set key. -->
+      <arg name="key_desc" type="s" direction="in" />
+      <!-- File descriptor through which to read the key data. -->
+      <arg name="key_fd" type="h" direction="in" />
+      <!--
+        b: True if the key state was changed, otherwise false.
+        b: True if the key was newly set in the keyring. False if a key with the
+           given key description already existed.
+      -->
+      <arg name="result" type="(bb)" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Start a pool. -->
+    <method name="StartPool">
+      <!-- UUID of pool to be started. -->
+      <arg name="pool_uuid" type="s" direction="in" />
+      <!-- Unlock method to use if pool is encrypted
+        b: True if an unlock method argument is being specified.
+        s: Method to use to unlock the pool, may be "keyring" or "clevis".
+      -->
+      <arg name="unlock_method" type="(bs)" direction="in" />
+      <!--
+        b: True if the pool was started, otherwise false.
+        o: Object path of the started pool.
+        ao: Array of object paths of block devices belonging to the pool.
+        ao: Array of object paths of filesystems belonging to the pool.
+      -->
+      <arg name="result" type="(b(oaoao))" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Stop a running pool. -->
+    <method name="StopPool">
+      <!-- Object path of pool to be stopped. -->
+      <arg name="pool" type="o" direction="in" />
+      <!--
+        b: True if the pool was stopped.
+        s: UUID of the stopped pool.
+      -->
+      <arg name="result" type="(bs)" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Unset a key in the kernel keyring. -->
+    <method name="UnsetKey">
+      <!-- Key description of the key to unset. -->
+      <arg name="key_desc" type="s" direction="in" />
+      <!-- b: True if the key was in the keyring, otherwise false. -->
+      <arg name="result" type="b" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!--
+      a{sa{sv}}: Mapping of Stratis pool UUIDs to device encryption information.
+        s: Stratis pool UUIDs
+        a{sv}: Mapping of device UUIDs to values.
+          s: Stratis device UUIDs
+          v: Encryption information for the device
+    -->
+    <property name="LockedPools" type="a{sa{sv}}" access="read" />
```

Comment:
I think this changed to `StoppedPools` in `r2`. I'm not sure how this ended up in the autogenerated output.